packages/deployer/src/compiler.ts

```diff
@@ -177,10 +177,15 @@ export class Compiler {
         );
 
         if (!_.isUndefined(compiled.errors)) {
+            const isError = (errorOrWarning: string) => !errorOrWarning.includes('Warning');
```

Comment:
Let's not have `Warning` be a string literal. Is these part of an error message we throw? If so, a constant should be used in both places linking them conceptually.

Reply:
It's a part of an error message solidity compiler throws. The only way to separate Warnings from errors is like that